config/paths.js

```diff
@@ -64,7 +64,7 @@ module.exports = {
   appJsConfig: resolveApp('jsconfig.json'),
   yarnLockFile: resolveApp('yarn.lock'),
   testsSetup: resolveModule(resolveApp, 'src/setupTests'),
-  proxySetup: resolveApp('src/setupProxy.js'),
+  proxySetup: resolveApp('dev_server/server.js'),
```

Comment:
moved setupProxy to dedicated server directory outside of `src`, assuming we'll continue to add features